drivers/cc2420/include/cc2420_internal.h

```diff
@@ -0,0 +1,250 @@
+/*
```

Comment:
Any reason why you need the `include` directory here? Putting it just into `drivers/cc2420/` would simplify the Makefiles a lot, I think.